src/components/list/UserBar/UserBar.tsx

```diff
@@ -69,7 +65,9 @@ function UserBar({ toggleAdminMode }: UserBarProps): JSX.Element {
 				title="Logout"
 			>
 				<Logout
-					handleLogout={handleLogout}
+					handleLogout={() => {
+						logout()
+					}}
```

Comment:
```suggestion
					handleLogout={logout}
```
Why would you write a function to run a function?